src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,4 +60,7 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+    public int compareTo(Name other) {
```

Comment:
Try to make use of java annotation. 
If you use the `@override` annotation, it will warn you in the event where you didn't override properly.